plasmapy/formulary/parameters.py

```diff
@@ -96,64 +97,101 @@ def _grab_charge(ion: Particle, z_mean=None):
 )
 def mass_density(
     density: [u.m ** -3, u.kg / (u.m ** 3)],
-    particle: Optional[Particle] = None,
-    z_mean: Optional[numbers.Real] = None,
+    particle: Union[Particle, str],
+    z_ratio: Optional[numbers.Real] = 1,
 ) -> u.kg / u.m ** 3:
-    """Utility function to merge two possible inputs for particle charge.
+    """
+    Calculated the mass density from a number density.
+
+    .. math::
+
+        \\rho = \\left| \\frac{Z_{s}}{Z_{particle}} \\right| n_{s} m_{particle}
+              = | Z_{ratio} | n_{s} m_{particle}
```

Comment:
I get how beautifully this renders, but I do a double take every time I look at the source... :sweat_smile: